### ISSUE TITLE
Riot web version shows as unknown in Riot settings

### ISSUE THREAD

**Issue by aaronraimist** [spantaleev/matrix-docker-ansible-deploy#26]

**Comment by spantaleev** [comment]
That's probably a problem with the [avhost/docker-matrix-riot](https://hub.docker.com/r/avhost/docker-matrix-riot/) image and how it builds Riot.

That image also has another problem that I'm hoping will get fixed soon - its incredibly huge size (~300+ MB).
If a better image comes along or if this one improves, I'd be happy to switch the playbook to it.
That's a separate issue though..

**Comment by aaronraimist** [comment]
Waiting on https://github.com/vector-im/riot-web/pull/7771

**Comment by spantaleev** [comment]
Looks like just what we need! Great find!

Let's switch to that, at least until vector-im/riot-web#7771 gets merged (which seems like something that won't happen any time soon).
